website/_sass/partials/_app.scss

```diff
@@ -15,21 +15,21 @@ body {
     [full-start] minmax(1rem, 1fr)
     [main-start] minmax(0, 52rem) [main-end]
     minmax(1rem, 1fr) [full-end];
-    background: rgb(14,23,42);
+    background: #0a101d;
+    // background: rgb(14,23,42) original background;
```

Comment:
Left the og background color here for future reference in case we need it. Its also the color of the horizon.